web/packages/design/src/Flex/Flex.story.jsx

```diff
@@ -21,11 +21,19 @@ import React from 'react';
 import Box from '../Box';
 
 import Flex from './Flex';
+import styled from 'styled-components';
 
 export default {
   title: 'Design/Flex',
 };
 
+const BoxWithBreakpoints = styled(Box)`
```

Comment:
About this part: this story was failing to render because our breakpoints are incorrectly defined; they don't have the `px` units embedded. I decided to leave it like this, as it's easier to circumvent it for now.